genrl/core/actor_critic.py

```diff
@@ -157,6 +172,8 @@ class CNNActorCritic(BaseActorCritic):
     def __init__(
         self,
         framestack: int,
+        action_dim: Space,
+        fc_layers: Tuple = (256,),
```

Comment:
`fc_layers` can be removed here. It's been substituted by `policy_layers` and `value_layers`